client/my-sites/activity/activity-log-item/index.jsx

```diff
@@ -388,7 +388,7 @@ class ActivityLogItem extends Component {
 						disableButton={ this.state.disableDownloadButton }
 					>
 						{ translate(
-							'{{time/}} is the selected point to create a download backup of. You will get a notification when the backup is ready to download.',
+							'{{time/}} is the selected point to create a download backup. You will get a notification when the backup is ready to download.',
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.